db/migrations/20240410141239-add-constraints-on-remediation-name.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+    up: async (q, {STRING}) => {
+        const transaction = await q.sequelize.transaction()
+        try {
+            // Add unique constraint to remediations name column based on tenant_org_id
+            await q.addConstraint('remediations', {
```

Comment:
This will fail if someone adds a record with a null or duplicate name between the time the previous migration ran and this migration runs - that's a pretty narrow window but it might be worth thinking about.  BTW I'm not sure the exact SQL this will generate so this might not be an issue, or we might need to include another run of what you did in the previous migration as part of this transaction.